features/step_definitions/register_page_steps.rb

```diff
@@ -0,0 +1,24 @@
+Given(/^I visit the register page$/) do
+  visit '/signup'
+end
+
+When(/^I fill in "([^"]*)" with "([^"]*)"$/) do |field_name, value|
+  fill_in field_name, with: value
+end
+
+When(/^I press the sign up button$/) do
+  click_button 'Sign up'
+end
+
+Then(/^I would like my information to be saved$/) do
+  user = User.last
+  expect(user.name).to eq 'Alfred'
```

Comment:
Can you figure out a better way to write this test rather than hard-coding `Alfred`

Reply:
FIxed